2.overview_and_terminology.md

```diff
@@ -145,6 +145,19 @@ metadata:
     name: foo
 ```
 
+*Okay:* Each group (`one.dev` and `other.dev`) allows the kind `Component` with name `foo`.
```

Comment:
What's the motivation to add this paragraph?

Reply:
It was not clear that a different group is enough. It is mentioned in the text but without example.